### PR TITLE
Commit 1

### PR DESCRIPTION
**Pull request by CheMBurN695**

### CODE REVIEW

File: Assets/Chemburn/PhysicsController.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+using System.Collections;
```
Comment (MatejMecka) [inline Assets/Chemburn/PhysicsController.cs]: Is this file needed?

File: Assets/Scripts/TestScript.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+using System.Collections;
```
Comment (MatejMecka) [inline Assets/Scripts/TestScript.cs]: What is this testing?

File: Assets/Chemburn/JumpController.cs [hunk 3]
```diff
@@ -0,0 +1,72 @@
+using System;
```
Comment (MatejMecka) [inline Assets/Chemburn/JumpController.cs]: Doesn't it make much more sense to place all the new scripts into the Script folder? Maybe divide them in folders: Player, Environment?

**Review by MatejMecka — CHANGES_REQUESTED** [review]

File: Assets/Chemburn/PhysicsController.cs (hunk 1 above)
Comment (CheMBurN695) [reply Assets/Chemburn/PhysicsController.cs]: oh I refactored to make it unecessary - to remove

File: Assets/Scripts/TestScript.cs (hunk 2 above)
Comment (CheMBurN695) [reply Assets/Scripts/TestScript.cs]: dont remember lol - to remove

File: Assets/Chemburn/JumpController.cs (hunk 3 above)
Comment (CheMBurN695) [reply Assets/Chemburn/JumpController.cs]: Yeah will do

File: Assets/Scripts/MoveController.cs [hunk 4]
```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+public class MoveController : MonoBehaviour
```
Comment (MatejMecka) [inline Assets/Scripts/MoveController.cs:6]:
```suggestion

public class MoveController : MonoBehaviour
```
